docs/EKS-cni-support.md

```diff
@@ -0,0 +1,22 @@
+# CNI monitoring special configuration updates for EKS
+
+AWS EKS uses [CNI](https://github.com/aws/amazon-vpc-cni-k8s) networking plugin for pod networking in Kubernetes using Elastic Network Interfaces on AWS
+
+One fatal issue that can occur is that you run out of IP addresses in your eks cluster. (Generally happens due to error configs where pods keep scheduling).
+
+You can monitor the `awscni` using kube-promethus with : 
+```
+local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
+    (import 'kube-prometheus/kube-prometheus-aws-eks-cni.libsonnet') +
```

Comment:
I didn't quite realize this before, but as we don't have an eks patch to begin with, I think we can rename this to be the "catch all" eks patch, as I assume this is the same for every eks cluster and all eks users want this.

Reply:
Hi @brancz sorry but I didn't quite get it.. do you mean to rename the patch file as `kube-prometheus-catch-all-eks.libsonnet` ?

Reply:
just `kube-prometheus-eks.libsonnet` :)

Reply:
13c114a72f4a7a24d2579dcd8aed15e76fd0b838 updated.